src/cljs/main/broadfcui/common/components.cljs

```diff
@@ -11,7 +11,6 @@
     [broadfcui.utils :as utils]
     ))
 
-
```

Comment:
re-add this line, it appears to be the only change in this file so let's not touch it